### PR TITLE
Adding ability to import dataset asset with no 'targets' field like filter divider

### PR DESCRIPTION
**Pull request by swish-gweisang**
Currently, importing a dashboard with filter dividers will break as the import of the dataset filters expects a 'targets' field in the yaml config file. However, a filter divider, which has no target, can still be imported if the 'targets' field defaults to an empty `dict`.

### CODE REVIEW

**Comment by Vitor-Avila** [comment]
Hey @swish-gweisang,

Thank you for catching this issue and also for working on a solution. It seems the latest commit removed your actual changes. Are you looking to move forward with this contribution? Happy to review the PR 🙌 🙏

File: src/preset_cli/cli/superset/sync/native/command.py [hunk 1]
```diff
@@ -326,7 +325,7 @@ def get_dataset_filter_uuids(config: AssetConfig) -> Set[str]:
     """
     dataset_uuids = set()
     for filter_config in config["metadata"].get("native_filter_configuration", []):
-        for target in filter_config["targets"]:
+        for target in filter_config.get("targets", dict()):
```
Comment (Vitor-Avila) [inline src/preset_cli/cli/superset/sync/native/command.py:328]: Could you replace this with:
``` python
        for target in filter_config.get("targets", {}):
```

I believe it's what's breaking CI and it's aligns with other occurrences in the repo.

Thank you! 🙏

**Comment by Vitor-Avila** [comment]
Hey @swish-gweisang, 

I left a comment in the PR. Also, could you add a test with a dashboard that would have a divider in its configuration? Or alternatively modify the dashboard mock used by tests that cover this flow to have a divider? This way we can prevent a regression.

Thank you!

**Comment by Vitor-Avila** [comment]
Hey @swish-gweisang,

I'm wondering if you had a chance to check my last message. I'm looking forward to push this one across the finish line. 🙌 

Thank you!

**Comment by Vitor-Avila** [comment]
thanks for adding the test @swish-gweisang! 🙏 I think we're only missing to resolve the conflicts and I can restart CI. Are you able to resolve the conflicts or would you like me to handle these? thanks!

**Comment by swish-gweisang** [comment]
Could you resolve these conflicts? 

Best
---
Guillaume Weisang, Ph. D.
Trading Analytics Manager
***@***.***

￼

**Review by Vitor-Avila — APPROVED** [review]
LGTM!

**Pull request merged**